pkg/reconciler/taskrun/hostupdate_test.go

```diff
@@ -92,18 +101,67 @@ var _ = Describe("HostUpdateTaskRunTest", func() {
 		}
 	})
 
+	It("should fail when the config doesn't exist", func(ctx SpecContext) {
+		k8sClient := fake.NewClientBuilder().
+			WithScheme(scheme).
+			WithInterceptorFuncs(interceptor.Funcs{
+				Get: func(
+					ctx context.Context,
+					client client.WithWatch,
+					key types.NamespacedName,
+					obj client.Object,
+					opts ...client.GetOption,
+				) error {
+					return errors.NewNotFound(schema.GroupResource{
+						Group:    obj.GetObjectKind().GroupVersionKind().Group,
+						Resource: "configmaps",
+					}, obj.GetName())
+				},
+			}).Build()
```

Comment:
We don't need this, as we are creating a new Client into which the ConfigMap have never been registered to

```suggestion
			Build()
```